seqr/views/apis/data_manager_api_tests.py

```diff
@@ -767,7 +836,7 @@ def _set_file_iter_stdout(rows):
                 self.assertEqual(response.status_code, 400)
                 self.assertDictEqual(response.json(), {'error': mock.ANY})
                 self.assertTrue(response.json()['error'].startswith(
-                    f'Error in NA19675_D2 data for {mismatch_row[2]}: mismatched entries '))
+                    f'Error in {loaded_data_row[0]} data for {mismatch_row[2]}: mismatched entries '))
```

Comment:
The `outlier` and `tpm` tests use `NA19675_D2` as the loaded sample id. But the `splice` can't use the same sample id because the `splice` data provide the individual id rather than the sample id, and there is no individual with `NA19675_D2` as the individual id in the fixture data. So the tests for `splice` data use a different sample id (`NA20889`).